src/backend/EdFi.DataManagementService.Backend.Postgresql/Operation/GetDocumentById.cs

```diff
@@ -52,7 +52,6 @@ NpgsqlTransaction transaction
                 return new GetResult.GetFailureNotExists();
             }
 
-            // TODO: Documents table needs a last modified datetime
```

Comment:
We have a ticket to deal with this, no need for a TODO comment (even if I'm the one who wrote it). The code analyzer was _sometimes_ (strangely, not _always_) complaining about these TODO comments.